Userland/Services/WindowServer/ScreenLayout.ipp

```diff
@@ -235,7 +235,16 @@ bool ScreenLayout::load_config(const Core::ConfigFile& config_file, String* erro
         auto group_name = String::formatted("Screen{}", index);
         if (!config_file.has_group(group_name))
             break;
-        screens.append({ config_file.read_entry(group_name, "Device"),
+        auto str_mode = config_file.read_entry(group_name, "Mode");
+        auto mode = str_mode == "Device" ? Screen::Mode::Device : str_mode == "Virtual" ? Screen::Mode::Virtual
+                                                                                        : Screen::Mode::Invalid;
+        if (mode == Screen::Mode::Invalid) {
```

Comment:
preexisting, but ... why is this file's extension .ipp?

Reply:
Probably include problems, idk 🤷